SConstruct

```diff
@@ -61,8 +61,9 @@ REPO_ROOT = GetOption("root")
 AddOption("--enable-profile", nargs="?", const="profile", dest="enable_profile",
           help=("Profile base filename; output will be <basename>-<sequence#>-<script>.pstats; "
                 "(Note: this option is for profiling the scripts, while --profile is for scons)"))
-AddOption("--butler-config", dest="butler_conf", default="",
-          help="Path to an external Butler config used to create a data repository.")
+AddOption("--butler-config", dest="butler_conf", default=os.path.join(PKG_ROOT, "configs", "registry.yaml"),
```

Comment:
Since this is a butler config argument and not a registry config, it's probably better to call this something like `butler-seed.yaml`